src/absil/il.fs

```diff
@@ -16,6 +16,7 @@ open System.Collections
 open System.Collections.Generic
 open System.Collections.Concurrent
 open System.Runtime.CompilerServices
+open System.Reflection
```

Comment:
for what do we need that?

Reply:
For the FieldAttributes, PropertyAttributes.... Enums

Reply:
If we don't want this dependency I can copy the enums

Reply:
we are ok with the dependency. The Fable build may need to duplicate the enum definition, hopefully @ncave will look after that on integration